KubeArmor/core/kubeArmor.go

```diff
@@ -505,6 +512,10 @@ func KubeArmor() {
 
 		// watch security policies
 		go dm.WatchSecurityPolicies()
+
+		// watch default posture
+		go dm.WatchDefaultPosture()
```

Comment:
move 516-517 below "dm.Logger.Print("Started to monitor security policies")"
add "dm.Logger.Print("Started to monitor per-namespace default posture")"